### PR TITLE
Removed unused favicon code as per #1064

### PR DESCRIPTION
**Pull request by codebu5ter**
All the code relating to the favicon has been removed.

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/web/IWebView.java [hunk 1]
```diff
@@ -109,12 +109,7 @@ public HitTarget(final boolean isLink, final String linkURL, final boolean isIma
     void restoreWebViewState(Bundle inState);
 
     void saveWebViewState(Bundle outState);
-
-    /**
-     * Get an icon (usually favicon) for the currently displayed website.
-     */
-    @Nullable
-    Bitmap getIcon();
+
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/web/IWebView.java]: \[Checkstyle\] ERROR: Trailing whitespace

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/web/IWebView.java]: \[Checkstyle\] ERROR: Line contains a tab character\.

File: app/src/webkit/java/org/mozilla/focus/webkit/FocusWebViewClient.java [hunk 2]
```diff
@@ -147,8 +145,7 @@ public void run() {
 
     @Override
     public void onPageStarted(WebView view, String url, Bitmap favicon) {
-        iconHolder.clearIcon();
-
+
```
Comment (sputnikci) [inline app/src/webkit/java/org/mozilla/focus/webkit/FocusWebViewClient.java]: \[Checkstyle\] ERROR: Trailing whitespace

Comment (sputnikci) [inline app/src/webkit/java/org/mozilla/focus/webkit/FocusWebViewClient.java]: \[Checkstyle\] ERROR: Line contains a tab character\.

File: app/src/main/java/org/mozilla/focus/web/IWebView.java (hunk 1 above)
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/web/IWebView.java]: \[Checkstyle\] ERROR: Trailing whitespace

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/web/IWebView.java]: \[Checkstyle\] ERROR: Line contains a tab character\.

File: app/src/webkit/java/org/mozilla/focus/webkit/FocusWebViewClient.java (hunk 2 above)
Comment (sputnikci) [inline app/src/webkit/java/org/mozilla/focus/webkit/FocusWebViewClient.java]: \[Checkstyle\] ERROR: Trailing whitespace

Comment (sputnikci) [inline app/src/webkit/java/org/mozilla/focus/webkit/FocusWebViewClient.java]: \[Checkstyle\] ERROR: Line contains a tab character\.

**Comment by codebu5ter** [comment]
@pocmo Can you please have a look at this again? :)
I've made a couple of commits, which should fix the issue.

**Comment by codebu5ter** [comment]
@pocmo  Done! :)

**Comment by pocmo** [comment]
@codebu5ter The PR somehow still shows 8 commits instead of one.

**Pull request closed without merging**